src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java

```diff
@@ -1299,10 +1299,15 @@ public void siAttendeCheLoStatoDellaNotificaSia(String statoNotifica) {
         piattaformaNotifichePage.selezionaNotificaConStato(statoNotifica);
     }
 
+    @And("Si attende completamento notifica {string}")
+    public void siAttendeCompletamentoNotificaV2(String statoNotifica) {
+        piattaformaNotifichePage.pollingSuStatoNotificaPerCompletamento(statoNotifica);
+    }
```

Comment:
delete  siAttendeCheLoStatoDellaNotificaDiventi method, eliminate where it is used and fix where needed

Reply:
Resolve